test/parallel/test-buffer-sharedarraybuffer.js

```diff
@@ -0,0 +1,25 @@
+/*global SharedArrayBuffer*/
+'use strict';
+// Flags: --harmony-sharedarraybuffer
+
+require('../common');
+const assert = require('assert');
+const Buffer = require('buffer').Buffer;
+
+const sab = new SharedArrayBuffer(24);
+const arr = new Uint16Array(sab);
+const ar = new Uint16Array(12);
```

Comment:
Just a suggestion, maybe it would be more readable to call them `arr1` and `arr2`?